### PR TITLE
EPMLSTRCMW-274 Create test implementations for Aggregation and Auth services

### PR DESCRIPTION
**Pull request by Nikolai-Isaev**
test: Create test implementations for Aggregation and Auth services

fix: Change to private takeUserFromRequest() method from AuthService

### CODE REVIEW

**Review by gominasian — APPROVED** [review]
Great!

**Review by DruzhininPavel — APPROVED** [review]
Looks good.

File: services/src/main/scala/cromwell/pipeline/service/AuthService.scala [hunk 1]
```diff
@@ -48,6 +46,15 @@ object AuthService {
 
       import authUtils._
 
+      private def takeUserFromRequest(request: SignInRequest): OptionT[Future, UserWithCredentials] =
```
Comment (myazinn) [inline services/src/main/scala/cromwell/pipeline/service/AuthService.scala]: Usually we place private methods at bottom

Comment (Nikolai-Isaev) [reply services/src/main/scala/cromwell/pipeline/service/AuthService.scala]: Thanks, fixed

**Review by Dmitrii-Shevchenko — APPROVED** [review]

**Review by myazinn — APPROVED** [review]

**Pull request merged**